main-support.ts

```diff
@@ -1,12 +1,26 @@
 import * as path from 'path';
+import mergeImg = require('merge-img');
```

Comment:
Unsure about imports, I think this is preferred:
```suggestion
import * as mergeImg from 'merge-img';
```